client/router_create.go

```diff
@@ -1118,6 +1181,14 @@ func (cli *VanClient) RouterCreate(ctx context.Context, options types.SiteConfig
 		ownerRefs = []metav1.OwnerReference{*siteOwnerRef}
 	}
 	var err error
+	isEdge := options.Spec.IsEdge()
+	if isEdge {
```

Comment:
Should this be restricted only to edge sites? (The inter-router-alias annotation would in that case be irrelevant).

Reply:
It shouldn't have, removed.